config/default.json

```diff
@@ -69,7 +69,8 @@
     },
     "refunds": "http://ccpay-refunds-api-prod.service.core-compute-prod.internal",
     "role_assignment": {
-      "roleApi": "http://am-role-assignment-service-prod.service.core-compute-prod.internal"
+      "roleApi": "http://am-role-assignment-service-prod.service.core-compute-prod.internal",
+      "roleMappingApi": "http://am-org-role-mapping-service-demo.service.core-compute-demo.internal"
```

Comment:
demo url in default.json needs to be changed to prod presumably?

Reply:
@MateiHMCTS  URL in prod always need to be pointing to PROD
You need to change it in your local-development or local-demo